spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ChannelBindingServiceProperties.java

```diff
@@ -174,4 +193,11 @@ public Properties getProducerProperties(String outputChannelName) {
 		}
 	}
 
+	public String getBinder(String channelName) {
```

Comment:
This class has evolved to include several of these "lookup" methods, rather than just playing the role of `@ConfigurationProperties`. Should we consider factoring some of that out?... perhaps as part of the larger changes such as removing `ChannelBindingService` altogether. Obviously, that would be beyond the scope of this PR.